README.md

```diff
@@ -23,3 +25,12 @@ Methods, post-ASpace transformation tools, and reports to support common SC acti
         ```
         BRANCH=<name> cap staging deploy
         ```
+     NB: You may need to run this with `bundle exec`
+     
+     NB: You may also need to run `bundle install --path vendor/bundle` first
+   1. cd to current on server
```

Comment:
Did we want to put the server name `lib-jobs-staging1` and `lib-jobs-prod1`

Reply:
can't hurt, right? (do we keep server names ever private?)

Reply:
No, we generally do not.